.github/workflows/stale-backtrace-issues.yml

```
@@ -0,0 +1,17 @@
+name: 'Manage stale backtrace reports'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          days-before-issue-stale: 60
+          days-before-issue-close: 1
+          only-issue-labels: 'backtrace.io'
+          stale-issue-label: 'stale-backtrace'
+          stale-pr-message: 'This issue is stale and will be closed tomorrow if no action is taken. To keep it open, leave a comment or remove the "stale" label.'
```

```suggestion
          stale-pr-message: 'This issue is stale and will be closed tomorrow if no action is taken. To keep it open, leave a comment or remove the `stale-backtrace` label.'
```


Can anyone leave a comment or just the developers?

---

Anyone, see the documentation about the action, its based on last update status of each issue/PR which comments update.